src/main/java/com/epam/reportportal/extension/bugtracking/InternalTicket.java

```diff
@@ -81,26 +81,63 @@ public void setBackLinks(Map<Long, String> backLinks) {
 	}
 
 	public static class LogEntry {
-		private Log log;
-		private InputStream attachment;
+		private Long logId;
+		private String message;
 		private boolean isIncludeLogs;
-
-		public LogEntry(Log log, InputStream attachment, boolean includeLogs) {
-			this.log = log;
-			this.attachment = attachment;
+		private boolean hasAttachment;
+		private String fileId;
+		private String decodedFileName;
+		private String contentType;
+		private boolean isImage;
+
+		public LogEntry(Long logId, String message, boolean includeLogs) {
+			this.logId = logId;
+			this.message = message;
 			this.isIncludeLogs = includeLogs;
 		}
 
-		public Log getLog() {
-			return log;
+		public LogEntry(Long logId, String message, boolean isIncludeLogs, boolean hasAttachment, String fileId, String decodedFileName,
+				String contentType) {
+			this.logId = logId;
+			this.message = message;
+			this.isIncludeLogs = isIncludeLogs;
+			this.hasAttachment = hasAttachment;
+			this.fileId = fileId;
+			this.decodedFileName = decodedFileName;
+			this.contentType = contentType;
+			this.isImage = ImageFormat.fromValue(MediaType.parse(contentType).getSubtype()).isPresent();
+		}
+
+		public Long getLogId() {
+			return logId;
 		}
 
-		public InputStream getAttachment() {
-			return attachment;
+		public String getMessage() {
+			return message;
 		}
 
-		public boolean getIncludeLogs() {
+		public boolean isIncludeLogs() {
 			return isIncludeLogs;
 		}
+
+		public boolean isHasAttachment() {
```

Comment:
isHas?

Reply:
it is needed for correct displaying on the "FTL" template